main.go

```diff
@@ -49,15 +45,20 @@ func main() {
 		restartThreshold := int32(NUMBER_RESTARTS)
 
 		for _, item := range pods.Items {
-		StatusLoop:
+			StatusLoop:
 			for _, status := range item.Status.ContainerStatuses {
 				waiting := status.State.Waiting
-
 				if waiting != nil {
 					reason := waiting.Reason
-					if stringInSlice(reason, waitingReasons) && status.RestartCount > restartThreshold {
-						fmt.Printf("%s / %s has %s restarts, which is over the %s restart limit.", item.Namespace,
-							item.Name, strconv.Itoa(int(status.RestartCount)), strconv.Itoa(int(restartThreshold)))
+					_, reasonInWaitingReasons := waitingReasons[reason]
+					if reasonInWaitingReasons || (reason == "CrashLoopBackOff" && status.RestartCount > restartThreshold) {
```

Comment:
I want to potentially brainstorm this if check. It makes sense to me to have "CrashLoopBackOff" in the map of reasons that are bad, but in that case I want to concisely capture these requirements for deletion:

- If the waiting reason is in the map and is anything but CrashLoopBackOff, then proceed with trying to delete
OR
- If the waiting reason is in the map AND is CrashLoopBackOff, then check if the number of restarts is over the threshold. If it is, then proceed with trying to delete

Any thoughts?